src/diffusers/models/resnet.py

```diff
@@ -34,21 +34,21 @@ def __init__(self, channels, use_conv=False, use_conv_transpose=False, out_chann
         else:
             self.Conv2d_0 = conv
 
-    def forward(self, hidden_states):
-        assert hidden_states.shape[1] == self.channels
+    def forward(self, input_tensor):
+        assert input_tensor.shape[1] == self.channels
```

Comment:
This shouldn't be changed.

Reply:
Hey @patil-suraj,
Thanks for pointing out the changes
those variables were recently changed in PR #676  
ill try to rename variables accordingly.